src/main/java/sirius/biz/jobs/batch/file/EntityExportJobFactory.java

```diff
@@ -80,6 +81,15 @@ protected void transferParameters(ImportContext context, ProcessContext processC
         // nothing to transfer by default
     }
 
+    /**
+     * Permits to provide a custom file name when exporting the entity.
+     *
+     * Otherwise the "end user friendly" plural of the entity is used as target file name
+     */
+    protected String setCustomFileName() {
```

Comment:
Ist das ned eher ne getCustomFileName oder check ich was ned ganz?

Reply:
Eigentlich ja... so ist auch in andere Methoden der Klasse und super-klassen wie getName, getLabel... habs geändert